### PR TITLE
[OBSDEF-1830] charts changes required for Telemetry collections and uploads

### PR DESCRIPTION
**Pull request by vgupta-mickey**
## Purpose
[OBSDEF-1830](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-1830)
The health framework will be used generate telemetry reports at objectscale-manager level and the same will be used to upload the reports at support-assist application level.
The customer and charts will provide the option to modify the followings:
Enable or Disable reports
Schedule
By using the health check framework, we can run these reports on demand basis in additon to the periodic reports.

## PR checklist
- [ x] make test passed
- [ x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: objectscale-manager/values.yaml [hunk 1]
```diff
@@ -237,3 +237,27 @@ healthChecks:
...
+      tag: # rfw-update-this objectscale-capacity-image
+    schedule: "0 * * * *"
+    disabled: true
+  performance:
+    image:
+      repository: objectscale-performance
+      tag: # rfw-update-this objectscale-performance-image
+    schedule: "*/10 * * * *"
+    disabled: true
+  healthReport:
```
Comment (eddiepavkovic) [inline objectscale-manager/values.yaml:258]:
```suggestion
  health:
```

**Review by eddiepavkovic — APPROVED** [review]

**Review by youminhan — APPROVED** [review]

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 2]
```diff
@@ -27,6 +27,30 @@ data:
           args:
             - -target-version
             - {{ default .Values.tag .Values.healthChecks.preUpdate.image.tag}}
+        - name: inventory
+          container: {{.Values.global.registry}}/objectscale-inventory:{{default .Values.tag .Values.healthChecks.inventory.image.tag}}
```
Comment (eddiepavkovic) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:31]:
```suggestion
          container: {{.Values.global.registry}}/{{ .Values.healthChecks.inventory.image.repository:{{default .Values.tag .Values.healthChecks.inventory.image.tag}}
```

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 3]
```diff
@@ -27,6 +27,30 @@ data:
...
+          serviceaccount: {{ .Release.Name }}-inventory
+          timelimit: "5m"
+          schedule: {{ default "0 * * * *" .Values.healthChecks.inventory.schedule | quote}}
+          disabled: {{ default false .Values.healthChecks.inventory.disabled }}
+        - name: capacity 
+          container: {{.Values.global.registry}}/objectscale-capacity:{{default .Values.tag .Values.healthChecks.capacity.image.tag}}
```
Comment (eddiepavkovic) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:37]: Please get the repo name from values file

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 4]
```diff
@@ -27,6 +27,30 @@ data:
...
+          serviceaccount: {{ .Release.Name }}-capacity
+          timelimit: "5m"
+          schedule: {{ default "0 * * * *" .Values.healthChecks.capacity.schedule | quote}}
+          disabled: {{ default false .Values.healthChecks.capacity.disabled }}
+        - name: performance 
+          container: {{.Values.global.registry}}/objectscale-performance:{{default .Values.tag .Values.healthChecks.performance.image.tag}}
```
Comment (eddiepavkovic) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:43]: get repo name from values file.

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 5]
```diff
@@ -27,6 +27,30 @@ data:
...
+          serviceaccount: {{ .Release.Name }}-performance
+          timelimit: "5m"
+          schedule: {{ default "*/5 * * * *" .Values.healthChecks.performance.schedule | quote}}
+          disabled: {{ default false .Values.healthChecks.performance.disabled }}
+        - name: health
+          container: {{.Values.global.registry}}/objectscale-health:{{default .Values.tag .Values.healthChecks.health.image.tag}}
```
Comment (eddiepavkovic) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:49]: get repo name from values file.

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml (hunk 4 above)
Comment (vgupta-mickey) [reply objectscale-manager/templates/objectscale-manager-app-configmap.yaml:43]: yep, cut and paste from healthcheck pre-update.

**Review by NguyenMartin — APPROVED** [review]

**Pull request merged**